docs/install.rst

```diff
@@ -237,11 +237,13 @@ Windows binaries
     Windows XP is not supported.
     Windows Vista requires at least SP2 to be installed. 
 
-You can download the latest stable Windows installer from the `GitHub Releases Page <https://github.com/streamlink/streamlink/releases/latest>`__.
+A Windows installer of the latest **stable release** can be found on the `GitHub releases page <https://github.com/streamlink/streamlink/releases/latest>`__.
 
-Alternatively, you can download the latest `nightly Windows installer <https://dl.bintray.com/streamlink/streamlink-nightly/streamlink-latest.exe>`__.
+Alternatively, a Windows installer of `latest development build <https://dl.bintray.com/streamlink/streamlink-nightly/streamlink-latest.exe>`__ for testing purposes is available,
```

Comment:
Maybe a link to the release notes on bintray could be added:
https://bintray.com/streamlink/streamlink-nightly/streamlink#release
Just a suggestion, because the changelog gets generated, but is never linked. I don't know though if a specific version can be permalinked. If not, a simple txt/md/rst file could be uploaded together with the installer. Then custom curl commands aren't even required and the files could be bundled in the regular json descriptor files.